pkg/graph/productAreas.resolvers.go

```diff
@@ -14,7 +14,17 @@ import (
 
 // Dataproducts is the resolver for the dataproducts field.
 func (r *productAreaResolver) Dataproducts(ctx context.Context, obj *models.ProductArea) ([]*models.Dataproduct, error) {
-	return r.repo.GetDataproductByProductArea(ctx, obj.ID)
+	teamsInPA, err := r.teamkatalogen.GetTeamsInProductArea(ctx, obj.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	teamIDsInPA := []string{}
+	for _, team := range teamsInPA {
+		teamIDsInPA = append(teamIDsInPA, team.ID)
+	}
+
```

Comment:
What about making line 17-26 GetTeam**IDs**InProductArea() function for teamkatalogen? it seems reusable for several places